README.md

```diff
@@ -1 +1,34 @@
-# hw01
\ No newline at end of file
+---
+output:
+  pdf_document: default
+  word_document: default
+  html_document: default
+---
+# Caleb Weis
+## Computing for the Social Sciences
+### Homework 1
+
+![Profile Picture](IMG_7332.JPG)
+
+Hey, I'm **Caleb Weis** (the one on the left--I'm 21, not 61!). I am a rising third-year in The College, and an Economics and Math major. I am interested in potentially doing economics research after I finish school--that is, if my professional acting career doesn't take off. I am an avid soccer player, and played competitively until the age of 19. My favorite professional soccer teams are:
+
+ * Manchester United
+ * Real Madrid
+ * New York City FC
+ * AC Milan
+ * Santos,
+ 
+and my favorite international soccer teams are:
+
+ 1. United States (patriotism!!)
```

Comment:
A very clear list. I have this thought that you meant to rank the teams because you use a different format when you list your favorite professional teams.